packages/web/src/js/Collapse.ts

```diff
@@ -108,7 +110,7 @@ class Collapse extends BaseComponent {
 
   updateCollapsibleElementHandler(collapsed: boolean = this.state.collapsed) {
     if (this.target) {
-      this.target.classList.toggle(CLASSNAME_COLLAPSED, collapsed);
+      this.target.classList.toggle(CLASSNAME_OPEN, collapsed);
```

Comment:
Please, pick up a better name for the internal variable `collapse`, eg. `is-open` !== `collapsed: true`. Because collapsed is a closed state, is it not?